extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/Descriptor.scala

```diff
@@ -151,7 +158,10 @@ case class FunctionDesc(
     val functionExtractor = functionExtractors(fieldExtractor)
     var function = functionExtractor(functionVal)
     if (function.database.isEmpty) {
-      function = function.copy(database = databaseDesc.map(_.extract(v)))
+      val maybeDatabase = databaseDesc.map(_.extract(v))
```

Comment:
irrelevant changes

Reply:
I think it's relevant as FunctionDesc using DataDesc, and now Database instead of pure string return from DataDesc

Reply:
I see